tests/testsuite/common.rs

```diff
@@ -38,9 +44,13 @@ pub fn render_module(module_name: &str) -> process::Command {
 
 /// Create a temporary directory with full access permissions (rwxrwxrwt).
 pub fn new_tempdir() -> io::Result<tempfile::TempDir> {
-    //  Using `tempfile::TempDir` directly creates files on macOS within
-    // "/var/folders", which provides us with restricted permissions (rwxr-xr-x)
-    tempfile::tempdir_in("/tmp")
+    if cfg!(not(windows)) {
+        //  Using `tempfile::TempDir` directly creates files on macOS within
+        // "/var/folders", which provides us with restricted permissions (rwxr-xr-x)
+        tempfile::tempdir_in("/tmp")
```

Comment:
Running some tests on another branch, I think we may no longer need this for tests to pass. Might've been solved by an update in `tempfile` or `docker`. Not too sure 🤷‍♂

Reply:
If it's still needed I think this should be changed to `cfg!(darwin)`